canonical-web-and-design/ubuntu.com
# Get the pending purchase status from the subscriptions.

## Done

- Get the pending purchase status from the subscriptions.
- Remove getting the pending purchase id from the window.

## QA

- Visit https://ubuntu.com/account/payment-methods?test_backend=true
- Change your card number to `4000000000000341`.
- Visit https://ubuntu.com/advantage?test_backend=true
- Open the edit form for a subscription and change the number of machines and click save.
- Visit https://ubuntu-com-10424.demos.haus/advantage?test_backend=true
- You should see a payment method warning at the top.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/278.
